core/include/moveit/task_constructor/storage.h

```diff
@@ -100,6 +100,8 @@ class InterfaceState
 
 		inline Status status() const { return std::get<0>(*this); }
 		inline bool enabled() const { return std::get<0>(*this) == ENABLED; }
+		inline bool failed() const { return std::get<0>(*this) == FAILED; }
+		inline bool pruned() const { return std::get<0>(*this) == DISABLED; }
```

Comment:
Shouldn't we use consistent naming here, i.e. either pruned or disabled?

Reply:
True, I propose to rename `DISABLED` to `PRUNED` for clarity. There is currently no other way to DISABLE a state.